packages/common/src/styles/ThemeProvider.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles';
+import { ThemeProvider as MuiThemeProvider } from '@material-ui/core';
```

Comment:
I don't know. Still experimenting, but importing everything I can from `@material-ui/core` seems to help ensure there are not multiple styles modules imported.